infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -262,6 +278,11 @@ func (b *TFBlueprintTest) ShouldSkip() bool {
 	return b.Spec.Skip
 }
 
+// ShouldRunTerraformVet checks if terraform vet should be executed
+func (b *TFBlueprintTest) ShouldRunTerraformVet() bool {
```

Comment:
Lets keep this private for now unless we have a need in the future.

Reply:
fixed